### DISCUSSION TITLE
Fix outstanding JavaFX bugs which make using SB annoying

### DISCUSSION THREAD

**Discussion by mipastgt** [gluonhq/scenebuilder#467] [General]
I think that before time is spent to plan new features for SB one should first fix the outstanding JavaFX bugs which make the use of SB quite annoying. E.g., even after 10 years it is still not possible to properly copy/paste a value into any of the SB text fields on Macs. Every string that you paste in there is always duplicated, so that you then have to manually delete half of the pasted content. (Just verified with the latest SB 17 on macOS 10.15.7)

https://bugs.openjdk.java.net/browse/JDK-8205915
https://bugs.openjdk.java.net/browse/JDK-8088897
I see there has been some activity recently. So let's keep our fingers crossed for JavaFX 19 and beyond.

What other bugs are there which should be fixed first?

**Comment by Oliver-Loeffler** [comment]
There is issue #373 which occurs on Linux.

It seems that is https://bugs.openjdk.java.net/browse/JDK-8269429 related but AFAIK there is no directly related issue in JBS.

**Comment by Oliver-Loeffler** [comment]
Another one is #357 - there is no JBS ticket assigned but also we do not have enough information about whats causing the issue.
The symptom is, the essentially the JavaFX GUI is rendered wrong. Could be a driver issue. I was not able to reproduce this at all.

**Comment by Oliver-Loeffler** [comment]
For Scene Builder a fix exists for JDK-8205915, which was introduced with PR #427. This came with Scene Builder 19.
I will look into [JDK-8088897](https://bugs.openjdk.java.net/browse/JDK-8088897) as well.

**Reply by Oliver-Loeffler** [reply]
Apparently [JDK-8205915](https://bugs.openjdk.java.net/browse/JDK-8205915) seems to be fixed at least as with OpenJFX18.
